gordo/server/views/base.py

```diff
@@ -268,7 +268,14 @@ def get(self, gordo_project: str):
         )
 
 
+class ExpectedModels(Resource):
+    @api.doc(description="Models that the server expects to be able to serve.")
+    def get(self, gordo_project: str):
+        return jsonify({"expected-models": current_app.config["EXPECTED_MODELS"]})
```

Comment:
It makes sense to me that this also reports the current version the server is running on.